android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/DayScheduleViewHolder.java

```diff
@@ -89,50 +94,34 @@ public void bindSession(RealmDataRepository realmRepo){
             slotTrack.getBackground().setColorFilter(storedColor, PorterDuff.Mode.SRC_ATOP);
             slotTrack.setText(sessionTrack.getName());
 
-            if(session.getIsBookmarked()) {
+            if (session.getIsBookmarked()) {
                 slot_bookmark.setImageResource(R.drawable.ic_bookmark_white_24dp);
             } else {
                 slot_bookmark.setImageResource(R.drawable.ic_bookmark_border_white_24dp);
             }
-            slot_bookmark.setColorFilter(storedColor,PorterDuff.Mode.SRC_ATOP);
+            slot_bookmark.setColorFilter(storedColor, PorterDuff.Mode.SRC_ATOP);
 
             final int sessionId = session.getId();
 
             slot_bookmark.setOnClickListener(v -> {
-                if(session.getIsBookmarked()) {
+                if (session.getIsBookmarked()) {
 
                     realmRepo.setBookmark(sessionId, false).subscribe();
                     slot_bookmark.setImageResource(R.drawable.ic_bookmark_border_white_24dp);
```

Comment:
Also, please change this to camel case while you're at it

Reply:
Can't find that method and if I refactor it here it treats it as a variable "isBookmarked" which is part of Session Realm Object

Reply:
I'm talking about `slot_bookmark` variable

Reply:
Done